CHANGELOG.md

```diff
@@ -12,6 +12,26 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Breaking changes
 
+### Deprecations
+
+### New features
+
+### Fixed bugs
+
+## [0.28.0] - 2025-04-02
+
+### Release highlights
+
+* All outstanding "pinned" issues in the GitHub repository have been resolved, including a long-standing bug relating to empty files in the built-in diff editor.
+
+* Many improvements to configuration management.
```

Comment:
(Optional) Perhaps "jj's configuration can now be split into multiple files more easily than before"?

Reply:
I've updated to that wording, but dropped "than before" because it seems redundant with "now" to me.